### PR TITLE
Product Details Component Creation 

### PR DESCRIPTION
**Pull request by stevehotcodes**
### The following PR has  the following tasks:
- Created the Product Details Component 
- Added the  app-product-details styles in the global stylesheet
- Corrected the products interface signature in the interface folder
- Created route for the products details in the app routing module

### CODE REVIEW

**Comment by JoshuaOndieki** [comment]
you should have switched branches by now. very misleading

**Comment by stevehotcodes** [comment]
I will switch in the next feature

**Comment by stevehotcodes** [comment]
But I believe the heading is well detailed for you to be cognizant with what I have done

**Comment by stevehotcodes** [comment]
Check whether everything is fine and let me know if there is any issue

File: src/styles.css [hunk 1]
```diff
@@ -116,3 +116,12 @@ app-signup,app-signin{
     align-items: center;
     margin: auto;
 }
+
+app-products-details{
+    border:2px solid red;
```
Comment (JoshuaOndieki) [inline src/styles.css]: remove debugging borders

**Review by JoshuaOndieki — CHANGES_REQUESTED** [review]

File: src/styles.css (hunk 1 above)
Comment (stevehotcodes) [reply src/styles.css]: done

**Review by JoshuaOndieki — APPROVED** [review]

**Pull request merged**